apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -24,15 +25,17 @@ export function toResponseWorkflowDto(
     user: preferences?.source[PreferencesTypeEnum.USER_WORKFLOW] || null,
     default: preferences?.source[PreferencesTypeEnum.WORKFLOW_RESOURCE] || DEFAULT_WORKFLOW_PREFERENCES,
   };
+  const workflowName = template.name || 'Missing Name';
 
   return {
     _id: template._id,
+    slug: `${slugifyName(workflowName)}_${encodeBase62(template._id)}`,
```

Comment:
remove slugifyName(workflowName), slug should be {prefix}{shortId} in order to support future refactoring, the client should structure the actual slug with {name}{slug}

```suggestion
    slug: `wf_${encodeBase62(template._id)}`,
```